ACT-reqs.md

```diff
@@ -0,0 +1,84 @@
+ACT Framework Requirements
+==========================
+
+Abstract
+--------
+
+The Accessibility Conformance Testing Framework will be a W3C specification, developed by the ACT Taskforce, lead by the WCAG Working Group. This document outlines the requirements for the ACT Framework. The ACT Framework is one part of the deliverables of the ACT Taskforce, for more details on this read [[ACT Deliverables]].
+
+Status of This Document
+-----------------------
+
+This is the first draft of this document, it has not yet had the full review of the ACT Taskforce. It is therefore likely to change significantly.
+
+
+1. Introduction
+---------------
+
+There are many accessibility test tools (ATTs). These can be fully automatic, or can prompt a user for specific actions, in order to discover accessibility issues of some digital product. Each of these tools will run one, or a serie of tests to come to some result. These test procedures may be designed to look for violations of WCAG 2.0, or some other accessibility standard, and for some that are highly customizable they can even be used in testing an organization's own accessibility policy. The test procedures used by ATT are often called 'rules'.
```

Comment:
sorry for picking on minor thing. "These can be **fully** automatic, or can prompt a user for specific actions, in order to discover accessibility issues of some digital product." Do we need "fully" for this? Is there anything **fully** automatic testing?

How we deal with acronym in the spec such as ATT?  Some other w3c doc I have seen has a link to the terminology including the definition.